microservices/recommendation/src/main/resources/application.yml

```
@@ -0,0 +1,101 @@
+eureka:
+  client:
+    serviceUrl:
+      defaultZone: http://${app.eureka-server}:8761/eureka/
+    initialInstanceInfoReplicationIntervalSeconds: 5
+    registryFetchIntervalSeconds: 5
+  instance:
+    leaseRenewalIntervalInSeconds: 5
+    leaseExpirationDurationInSeconds: 5
+
+spring.data.mongodb:
+      auto-index-creation: true
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 6

(indentation)